services/gateway/sources/version3/mappings/token.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	tokenID: '=,string',
+	name: '=,string',
+	symbol: '=,string',
+	availableBalance: '=,string',
+	lockedAmount: ['lockedBalances', {
```

Comment:
Please clarify this with the FE team and update the field name to be the same as `lockedBalances`. Please update the issue and tech spec document after confirmation.

Reply:
Clarified with the FE team and updated!